linkerd/app/gateway/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "linkerd2-app-gateway"
+version = "0.1.0"
+authors = ["Linkerd Developers <cncf-linkerd-dev@lists.cncf.io>"]
+edition = "2018"
+publish = false
+
+[dependencies]
+http = "0.1"
```

Comment:
Why does this depend on version 0.1 of `http`? The proxy should be using `http` 0.2 more or less everywhere else.